client/pages/login-form.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      dueDate: '',
+      babyName: '',
+      password: ''
+    };
+    this.handleEmail = this.handleEmail.bind(this);
+    this.handleDueDate = this.handleDueDate.bind(this);
+    this.handleBabyName = this.handleBabyName.bind(this);
+    this.handlePassword = this.handlePassword.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleEmail(event) {
+    this.setState({
+      email: event.target.value
+    });
+  }
+
+  handleDueDate(event) {
+    this.setState({
+      dueDate: event.target.value
+    });
+  }
+
+  handleBabyName(event) {
+    this.setState({
+      babyName: event.target.value
+    });
+  }
+
+  handlePassword(event) {
+    this.setState({
+      password: event.target.value
+    });
+  }
```

Comment:
You can simplify this to one method with:
```JSX
handleChange(event) {
  const name = event.target.name;
  const value = event.target.value;
  
  this.setState({ [name]: value });
}
```

Reply:
@scbowler 
Thanks for the comment. Code updated.